posts/2022-05-12-stacktrace-log-22006-beta.adoc

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "New Logging functionality to join stack traces in logs in Open Liberty 22.0.0.6-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: New Logging functionality to join stack traces in logs in Open Liberty 22.0.0.6-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.6-beta offers one new beta feature (along with all GA features), which provides new logging functionality to join stack traces in logs, allowing for the stack trace emitted to be output as a single log event.
+blog_description: Open Liberty 22.0.0.6-beta offers one new beta feature (along with all GA features), which provides new logging functionality to join stack traces in logs, allowing for the stack trace emitted to be output as a single log event.
```

Comment:
```suggestion
blog_description: Open Liberty 22.0.0.6-beta offers one new beta feature (along with all GA features), which provides new logging functionality to join stack traces in logs so that they are output as a single log event.
```